cordapp/src/test/kotlin/net/corda/plugins/CordappTest.kt

```diff
@@ -47,6 +48,7 @@ class CordappTest {
         val jarTaskRunner = jarTaskRunner("CorDappWithInfo.gradle", extraArgs)
 
         val result = jarTaskRunner.build()
+        println(result.output)
```

Comment:
Maybe this `println` can be removed

Reply:
I added those `println()` deliberately so that the output from the test build becomes visible in the JUnit output :grinning:.